administrator/language/en-GB/plg_sampledata_blog.ini

```diff
@@ -12,12 +12,15 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_0_TITLE="About"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_1_TITLE="Working on Your Site"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_10_FULLTEXT=""
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_10_INTROTEXT="<div><h1>Typography in Cassiopeia(h1)</h1><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h2>Lorem ipsum dolor sit amet (h2)</h2><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h3>Lorem ipsum dolor sit amet (h3)</h3><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h4>Lorem ipsum dolor sit amet (h4)</h4><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h5>Lorem ipsum dolor sit amet (h5)</h5><p> </p><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><p> </p><a href=\"link\"> Lorem ipsum dolor (a)</a></p><p><span class=\"btn btn-primary\">primary</span><span class=\"btn btn-secondary\">secondary</span><span class=\"btn btn-info\">info</span><span class=\"btn btn-success\">success</span><span class=\"btn btn-warning\">warning</span><span class=\"btn btn-danger\">danger</span></p><p><span class=\"text text-info\">text-info</span>  <span class=\"text text-danger\">text-danger</span>  <span class=\"text text-primary\">text-primary</span>  <span class=\"text text-secondary\">text-secondary</span></p><h4>(ol)(li)</h4><ol><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ol><p> </p><h4>(ul)(li)</h4><ul><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ul><p class=\"display-1\">Lorem (display-1)</p><p class=\"display-2\">Lorem (display-2)</p><p class=\"display-3\">Lorem ipsum (display-3)</p><p class=\"display-4\">Lorem ipsum (display-4)</p></div>"
```

Comment:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_10_INTROTEXT="<div><h1>Typography in Cassiopeia (h1)</h1><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h2>Lorem ipsum dolor sit amet (h2)</h2><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h3>Lorem ipsum dolor sit amet (h3)</h3><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h4>Lorem ipsum dolor sit amet (h4)</h4><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h5>Lorem ipsum dolor sit amet (h5)</h5><p> </p><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><p> </p><a href=\"link\"> Lorem ipsum dolor (a)</a></p><p><span class=\"btn btn-primary\">primary</span><span class=\"btn btn-secondary\">secondary</span><span class=\"btn btn-info\">info</span><span class=\"btn btn-success\">success</span><span class=\"btn btn-warning\">warning</span><span class=\"btn btn-danger\">danger</span></p><p><span class=\"text text-info\">text-info</span>  <span class=\"text text-danger\">text-danger</span>  <span class=\"text text-primary\">text-primary</span>  <span class=\"text text-secondary\">text-secondary</span></p><h4>(ol)(li)</h4><ol><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ol><p> </p><h4>(ul)(li)</h4><ul><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ul><p class=\"display-1\">Lorem (display-1)</p><p class=\"display-2\">Lorem (display-2)</p><p class=\"display-3\">Lorem ipsum (display-3)</p><p class=\"display-4\">Lorem ipsum (display-4)</p></div>"
```
Add missing space before the "(h1)" to be consistent with the other "(...)".

Reply:
thank you, done